calendar/engine/daily_summary.go

```diff
@@ -291,3 +292,17 @@ func getTodayHoursForTimezone(now time.Time, timezone string) (start, end time.T
 	end = start.Add(24 * time.Hour)
 	return start, end
 }
+
+func convertToUpperCaseTime(timeStr string) string {
```

Comment:
Can we rename this to reflect that it is only changing the case for the meridiems?

Reply:
Sure, done @wiggin77 . 
Updated it to `convertMeridiemToUpperCase`